go.mod

```diff
@@ -1,5 +1,5 @@
 module github.com/urfave/cli/v3
-
+module github.com/codegangsta/cli
```

Comment:
@pankaj1879 Why do we need this ? the codegangsta/cli corresponds to v1 of this module.

Reply:
facing this error 
github.com/codegangsta/cli: github.com/codegangsta/cli@v1.22.10: parsing go.mod:
	module declares its path as: github.com/urfave/cli
	        but was required as: github.com/codegangsta/cli


please help me

Reply:
@pankaj1879 You need to update your go.mod as follows

```
replace github.com/codegangsta/cli v1.22.10 => github.com/urfave/cli
```